modules/ROOT/pages/release-notes/release-notes57.adoc

```diff
@@ -151,23 +154,23 @@ For information on the Spell Checker Pro API, see: link:{baseurl}/plugins/premiu
 * `mceSpellcheckDialog`
 * `mceSpellcheckDialogClose`
 
-For information on the Spell Checker Pro Commands, see: link:{baseurl}/plugins/premium/tinymcespellchecker/#commands[Spell Checker Pro - Commands].
+For information on the Spell Checker Pro Commands, see: xref:plugins/premium/tinymcespellchecker.adoc#commands[Spell Checker Pro - Commands].
 
 ===== New Spell Checker Pro Event
 
 *Spell Checker Pro* introduces a new `SpellcheckerLanguageChanged` event which is fired when the active language is changed.
 
-For information on the Spell Checker Pro Events, see: link:{baseurl}/plugins/premium/tinymcespellchecker/#events[Spell Checker Pro - `Events`].
+For information on the Spell Checker Pro Events, see: xref:plugins/premium/tinymcespellchecker.adoc#events[Spell Checker Pro - `Events`].
 
 ==== Spell Checker Pro - Enhancements
 
 *Spell Checker Pro* 2.3.0 provides the following enhancements:
 
 ===== The `spellchecker_ignore_list` option now accepts arrays of words for specific languages
 
-*Spell Checker Pro* 2.3.0 introduces enhancements to the `spellchecker_ignore_list` (formerly `spellchecker_whitelist`, see: <<thespellchecker_whitelistoptionhasbeenrenamed,The `spellchecker_whitelist` option has been renamed>>).
+*Spell Checker Pro* 2.3.0 introduces enhancements to the `spellchecker_ignore_list` (formerly `spellchecker_whitelist`, see: xref:thespellchecker_whitelistoptionhasbeenrenamed[The `spellchecker_whitelist` option has been renamed]).
 
-It is now possible to specify arrays of words for specific languages to be ignored by the Spell Checker Pro plugin using the `spellchecker_ignore_list` option, see: link:{baseurl}/plugins/premium/tinymcespellchecker/#spellchecker_ignore_list[Spell Checker Pro - `spellchecker_ignore_list`].
+It is now possible to specify arrays of words for specific languages to be ignored by the Spell Checker Pro plugin using the `spellchecker_ignore_list` option, see: xref:plugins/premium/tinymcespellchecker.adoc#spellchecker_ignore_list[Spell Checker Pro - `spellchecker_ignore_list`].
```

Comment:
This xref doesn't resolve as the `spellchecker_ignore_list` id doesn't exist in `modules/ROOT/partials/configuration/`spellchecker_ignore_list.adoc`. It needs to be added to the option heading, e.g:
```
[[spellchecker_ignore_list]]
=== `spellchecker_ignore_list`
```

Reply:
done